opacus/accountants/utils.py

```diff
@@ -13,26 +13,19 @@
 # limitations under the License.
 
 from opacus.accountants import create_accountant
+from typing import Optional
 
-
-# min range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MIN_BOUND = 0.01
-# starting point for a max range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MAX_BOUND = 10
-# condition to halt binary search for sigma given epsilon
-SIGMA_PRECISION = 0.01
-# max possible value for returned sigma.
-# Noise higher than MAX_SIGMA considered unreasonable
-MAX_SIGMA = 2000
-
+MAX_SIGMA = 1e6
```

Comment:
There was a comment before which I recommend to leave because the constant looks a bit surprising.

```python
# Noise level higher than MAX_SIGMA considered unreasonable 
# leading to "not sufficient privacy budget" exception
MAX_SIGMA = 1e6
```